lib/main.dart

```diff
@@ -124,8 +124,20 @@ class _AppState extends State<App> {
   bool _isDiscoModeRunning = false;
   Timer? _discoModeTimer;
 
+  void spawnIsolate(AppLocalizations localizations) async {
+    await tts_isolate.isolateMain(ReceivePort().sendPort,
+        channelStreamController, widget.prefs, localizations);
+  }
+
   @override
   Widget build(BuildContext context) {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
It might be better to fetch app localizations in the isolate instead of stealing it from the context. This might result it multiple isolates starting

Reply:
Ok I think I handled that though I need a better variable name than locale

Reply:
Looks good